lib/url.js

```diff
@@ -958,7 +958,8 @@ Url.prototype.resolveObject = function resolveObject(relative) {
     srcPath.unshift('');
   }
 
-  if (hasTrailingSlash && (srcPath.join('/').substr(-1) !== '/')) {
+  const srcPathSlashes = srcPath.join('/');
+  if (hasTrailingSlash && (srcPathSlashes.substring(srcPathSlashes.length - 1) !== '/')) {
```

Comment:
shouldn't this be `srcPathSlashes.charAt(-1)` instead of `srcPathSlashes.substring(srcPathSlashes.length - 1)`?

Reply:
`srcPathSlashes.charAt(srcPathSlashes.length -1)`

Reply:
> `srcPathSlashes.charAt(srcPathSlashes.length -1)`

hmm I mean sure... but `.charAt(-1)` accomplishes the same result

Reply:
Err.. actually just `.slice(-1)`

Reply:
`.charAt(-1)` works differently than before. I think `.slice(-1)` is correct for this case. Thank you for your opinion.